openfisca_us/variables/states/ca/calepa/carb/cvrp/is_ca_cvrp_increased_rebate_eligible.py

```diff
@@ -0,0 +1,23 @@
+from openfisca_us.model_api import *
+
+
+class is_ca_cvrp_increased_rebate_eligible(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    label = "Eligible for CVRP increased rebate"
+    documentation = "Eligible for increased rebate for low- and middle-income participants in California's Clean Vehicle Rebate Project (CVRP)"
+
+    def formula(person, period, parameters):
+        # Use school meal FPG ratio.
+        fpg_ratio = person.spm_unit("school_meal_fpg_ratio", period)
+        p = parameters(period).states.ca.calepa.carb.cvrp.increased_rebate
+        income_eligible = fpg_ratio <= p.fpl_limit
+        categorically_eligible = np.any(
```

Comment:
Yeah we should add a pattern for this in `openfisca-tools`.

Reply:
Filed https://github.com/PolicyEngine/openfisca-tools/issues/22